src/Symfony/Component/Messenger/Exception/ThrownExceptionDetails.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Symfony\Component\Messenger\Exception;
+
+use Symfony\Component\ErrorHandler\Exception\FlattenException;
+
+/**
+ * ThrownExceptionDetails captures details of a thrown exception and is displayed like a normal exception when dumped
+ * with the VarDumper.
+ *
+ * @author Jeroen Noten <jeroennoten@me.com>
+ */
+class ThrownExceptionDetails
+{
+    private $class;
+    /**
+     * @var string
+     */
+    private $message;
+    /**
+     * @var int
+     */
+    private $code;
+    /**
+     * @var string
+     */
+    private $file;
+    /**
+     * @var int
+     */
+    private $line;
+    /**
+     * @var array
```

Comment:
I'd suggest removing all annotations and replace them with default values.

Reply:
Class is removed.